gapic-generator/lib/gapic/schema/api.rb

```diff
@@ -198,6 +200,29 @@ def configuration
         end
       end
 
+      # Raw parsed json of the grpc service config if provided 
+      # or an empty hash if config was not provided
+      def grpc_service_config_raw
+        @grpc_service_config_raw ||= begin
+          grpc_service_config_filename = protoc_options["grpc_service_config"]
+          if grpc_service_config_filename
+            file = ::File.read grpc_service_config_filename
+            JSON.parse file
+          else
+            {}
+          end
+        end
+
+        @grpc_service_config_raw
```

Comment:
This line is not needed. The previous clause already returns `@grpc_service_config_raw`.